src/core/api/io_services/v1/post_createservice_policy/policy.xml

```diff
@@ -0,0 +1,20 @@
+<policies>
+    <inbound>
+        <!--Service creation implies subscription creation. 
+            There is an hard-limit o APIM such as creations cannot happen concurrently on the same product
+            (because every subscription creation modifies the arm resource).
+            In order to mitigate so, we limit concurrency ourself.-->
+        <limit-concurrency key="@(context.User.Id)" max-count="5">
```

Comment:
These policy groups limitations by the same user id; each user can have at most 5 concurrent `createService` open requests. 

As a matter of fact, there's no reason to count for the same user as different users would be affected by the bug anyway. 

Should I use a all-user key?